tests/functional/replication/queueProcessor.js

```diff
@@ -34,15 +34,38 @@ const constants = {
     },
     key: 'key_to_replicate',
     versionId: '98498980852335999999RG001  100',
-    body: 'some contents to be replicated',
+    partOneBody: 'some contents to be replicated',
+    partTwoBody: 'some more contents to be replicated',
     roleSessionName: 'backbeat-replication',
 };
 
+const partsContent = [constants.partOneBody, constants.partTwoBody];
```

Comment:
Those two should be part of the `S3Mock` object, built in the constructor (the idea is being able to spawn multiple `S3Mock` objects at the same time with different params).